src/shared/converters/image.js

```diff
@@ -40,45 +39,67 @@ export default {
           eventAction="open content media"
         >
           <a>
-            <Image content key={attachmentId} id={attachmentId} />
+            <Image isContent key={attachmentId} id={attachmentId} />
           </a>
         </Link>
       );
     }
 
-    let src;
+    const alt = attributes.alt || null;
 
-    // Get src attribute from different cases or assign an empty string.
+    let src = null;
+
+    // Get src attribute from different cases.
     if (attributes.src && typeof attributes.src === 'string') {
-      ({ src } = attributes);
+      if (attributes.src.startsWith('http')) src = he.decode(attributes.src);
+      else src = he.decode(`${settings.generalSite.url}${attributes.src}`);
     } else if (
       attributes.dataset &&
       attributes.dataset.original &&
       typeof attributes.dataset.original === 'string'
     ) {
-      src = attributes.dataset.original;
-    } else {
-      src = '';
+      if (attributes.src.startsWith('http'))
+        src = he.decode(attributes.dataset.original);
+      else
+        src = he.decode(
+          `${settings.generalSite.url}${attributes.dataset.original}`,
+        );
+    }
+
+    let srcSet = null;
+
+    // Get srcset attribute from different cases.
+    if (attributes.srcset && typeof attributes.srcset === 'string') {
+      srcSet = attributes.srcset
+        .split(',')
+        .map(s => {
+          const trimmed = s.trim();
+
+          if (trimmed.startsWith('http')) return trimmed;
+
+          return he.decode(`${settings.generalSite.url}${trimmed}`);
```

Comment:
We can also take the opportunity to remove `he.decode`.

Reply:
Sure, will do.